.github/workflows/post-promote-tasks.yml

```diff
@@ -71,13 +79,13 @@ jobs:
               {
                 "releaseName": "${{ matrix.amp-version }}",
                 "time": "$(date +'%Y-%m-%d %H:%M:%S %Z')",
-                "channel": "${{ matrix.calendar-channel }}"
+                "channel": "${{ matrix.channel }}"
               }
             }
 
   create-issue-on-error:
     if: failure()
-    needs: [setup, tasks]
+    needs: [setup, npm-publish, release-calendar]
```

Comment:
I feel like I might've asked this before so forgive this uncaffeinated comment if I already have: did we verify what happens with `create-issue-on-error` when all 3 `needs` are skipped? When 2 are skipped and 1 fails? ... 1 passes? etc...